__tests__/app.test.js

```diff
@@ -44,16 +44,66 @@ describe("GET /api/articles",() => {
         })
     })
 })
-
+//article comments
+describe("GET /api/articles/:article_id/comments", () => {
+    test("200: responds with an array of comments that relate to the chosen article", () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body: { comments }}) => {
+            expect(comments).toHaveLength(11)
+            comments.forEach((comment) => {
+                expect(comment.article_id).toBe(1)
+                expect(comment).toMatchObject({
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),})
+            })
+        })
+    })
+    test("200: responds with empty array if article exists but has no comments",() => {
+        return request(app)
+        .get('/api/articles/2/comments')
+        .expect(200)
+        .then(({ body: { comments }}) => {
+            expect(comments).toEqual([])
+        })
+    })
+    test("status: 200, most recent comment is served first", () => {
```

Comment:
This can be combined into your test on line 49 if you wanted to. In integration testing we are testing the full server response in one. Unlike unit testing where I would tell you to test each bit separately.